contracts/factory/K1ValidatorFactory.sol

```diff
@@ -63,11 +63,7 @@ contract K1ValidatorFactory is Stakeable {
         IERC7484 registry
     ) Stakeable(factoryOwner) {
         require(
-            !(implementation == address(0) ||
-                k1Validator == address(0) ||
-                address(bootstrapper) == address(0) ||
-                address(registry) == address(0) ||
-                factoryOwner == address(0)),
+            !(implementation == address(0) || k1Validator == address(0) || address(bootstrapper) == address(0)),
```

Comment:
why this is skipped too?
factoryOwner == address(0)

Reply:
Weird, it was not removed on my local version